api/schemas/report.json

```diff
@@ -114,6 +114,11 @@
             ]
         },
 
+        "master": {
+            "description": "The name of the master that was used to compile the catalog. If failover occurred, this will hold the first master successfully contacted. If this run had no master (e.g. a `puppet apply` run), this field will be blank",
```

Comment:
I'm not actually sure what happens if no master can be contacted because of an error, because when I tried it no report was generated. There's a chance it would just be ":" because of the string interpolation, but I'm not sure how to test it.

If we just have the server setting and not a list, how do we get the port?